README.md

```diff
@@ -6,7 +6,7 @@
 
 Alfa is an open and standards-based accessibility conformance testing engine used for testing websites built using HTML, CSS, and JavaScript against accessibility standards such as the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG/). It is the result of distilling the best parts of Siteimprove's proprietary accessibility conformance testing engine and implementing them on top of the open [Accessibility Conformance Testing (ACT) Rules Format](https://www.w3.org/TR/act-rules-format/). In comparison to Siteimprove's proprietary engine, Alfa also brings several improvements that make it possible to implement and execute advanced rules without relying on Siteimprove infrastructure.
 
-> :warning: Alfa is still in the very early stages of development. Nothing is final, nothing is published, and breaking API changes will be swift and unforgiving. You should however not let it deter you from exploring the project and some of the new and interesting ideas we're trying to bring to the table.
+> :warning: Alfa is still in the very early stages of development. Nothing is final, nothing is published, and breaking API changes are swift and unforgiving. You should however not let it deter you from exploring the project and some of the new and exciting ideas we're trying to bring to the table.
```

Comment:
I preferred the old version

Reply:
I changed the tense of the breaking API changes, as breaking API changes has already happened. I replaced `interesting` as it is often overused.

Reply:
Agree on the former, disagree on the latter; playing the synonyms game is a smell if anything. There's nothing wrong with removing it altogether though:

> [...] some of the new ideas we're trying to bring to the table.